crates/feldera-types/src/transport/postgres.rs

```diff
@@ -21,4 +21,16 @@ pub struct PostgresWriterConfig {
 
     /// The table to write the output to.
     pub table: String,
+
+    /// The CA certificate in PEM format.
+    pub ssl_ca_pem: Option<String>,
+
+    /// The client certificate in PEM format.
+    pub ssl_client_pem: Option<String>,
```

Comment:
for the whole section, are these secrets or just path refs? how do we avoid this getting shared in logs/printing connector config etc.

I believe there is already infra for this, check with @blp